src/SprykerEco/Client/Billpay/Zed/BillpayStub.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Client\Billpay\Zed;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Client\ZedRequest\Stub\ZedRequestStub;
+
+class BillpayStub extends ZedRequestStub implements BillpayStubInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Spryker\Shared\Kernel\Transfer\TransferInterface | \Generated\Shared\Transfer\BillpayPrescoringTransactionResponseTransfer
```

Comment:
invalid doc blocks, please check code sniffer, no whitespace on piping.

Also never interface on transfers, always concrete one \Generated\Shared\Transfer\BillpayPrescoringTransactionResponseTransfer

Reply:
Make sure it also adheres to phpstan level 5, you might need https://github.com/spryker/spryker/wiki/PHPStan-notes-tips#stubs